src/test/java/teammates/common/util/FieldValidatorTest.java

```diff
@@ -589,6 +591,26 @@ public void testGetInvalidityInfoForTimeForVisibilityStartAndResultsPublish_inva
                          visibilityStart, resultsPublish));
     }
 
+    @Test
+    public void testGetInvalidityInfoForTimeForSessionEndAndExtendedDeadlines_valid_returnEmptyString() {
+        Instant sessionEnd = TimeHelperExtension.getInstantHoursOffsetFromNow(-1);
+        Map<String, Instant> extendedDeadlines = new HashMap<>();
+        extendedDeadlines.put("participant@email.com", TimeHelperExtension.getInstantHoursOffsetFromNow(1));
+        assertEquals("",
+                FieldValidator.getInvalidityInfoForTimeForSessionEndAndExtendedDeadlines(
+                        sessionEnd, extendedDeadlines));
+    }
+
+    @Test
+    public void testGetInvalidityInfoForTimeForSessionEndAndExtendedDeadlines_invalid_returnErrorString() {
+        Instant sessionEnd = TimeHelperExtension.getInstantHoursOffsetFromNow(1);
+        Map<String, Instant> extendedDeadlines = new HashMap<>();
+        extendedDeadlines.put("participant@email.com", TimeHelperExtension.getInstantHoursOffsetFromNow(-1));
+        assertEquals("The extended deadlines for this feedback session cannot be earlier than the end time.",
+                FieldValidator.getInvalidityInfoForTimeForSessionEndAndExtendedDeadlines(
+                        sessionEnd, extendedDeadlines));
+    }
+
```

Comment:
Just wondering: what will the case be if the deadline is exactly the same as the session deadline? 
To me it shouldn't be created too, as it is redundant.

Reply:
It adds them right now, but you're right they should not create them. I will change it.